config/haproxy_template.cfg

```diff
@@ -53,40 +57,57 @@ frontend http-in
         stats auth admin:admin
         stats uri /haproxy_stats
 
-{{ range $index, $app := .Apps }} {{ if $app.Env.BAMBOO_TCP_PORT }}
-listen {{ $app.EscapedId }}-cluster-tcp :{{ $app.Env.BAMBOO_TCP_PORT }}
-        mode tcp
-        option tcplog
-        balance roundrobin
-        {{ range $page, $task := .Tasks }}
-        server {{ $app.EscapedId}}-{{ $task.Host }}-{{ $task.Port }} {{ $task.Host }}:{{ $task.Port }} {{ if $app.HealthCheckPath }} check inter 30000 {{ end }} {{ end }}
-{{ end }}
+frontend https-in
+        bind *:443 ssl crt /etc/ssl/dataman.io.pem
```

Comment:
It's probably not appropriate to have the ssl cert paths of a specific organisation in our default template